hosting_docs/source/security/1-security.rst

```diff
@@ -1,4 +1,65 @@
 Securing CommCareHQ
 ===================
 
-Notes on implementing all of security best practices when hosting HQ
\ No newline at end of file
+Security is once of the most important things when dealing with web applications. This guide gives high level overview of security considerations to protect the data that is collected using your CommCareHQ instance. Note that this just gives hints in the direction of the security and is in no way exhaustive.
+
+Introduction
+------------
+
+A web application exposed to public internet has many levels where an attack is possible such as application, host operating system, network and even physical levels. In a Cloud environment, the cloud service provider handles the security upto operating system. But when hosting on premises all of this responsiblity falls upon the team hosting the application. 
+
+It is very important to design an end-to-end hosting architecture and processes to maximize security at all levels. In addition, there might be government regulations and legal requirements to be met when hosting locally that you may have to know.
+
+Below are few architecture references from the industry to help you understand number of considerations when designing a secure hosting environment.
+
+#. https://www.cisco.com/c/dam/en/us/solutions/collateral/enterprise/design-zone-security/safe-secure-dc-architecture-guide.pdf
+#. https://www.oracle.com/a/ocom/docs/oracle-cloud-infrastructure-security-architecture.pdf
+#. https://nvlpubs.nist.gov/nistpubs/legacy/sp/nistspecialpublication800-123.pdf
+
+Below we provide few of such considerations that we recommend at a minimum.
```

Comment:
@proteusvacuum @Charl1996  Please let me know if there are any additional things that we should add here.